diff --git a/test/cctest/cctest.gyp b/test/cctest/cctest.gyp
index a1f49b9..a48eb4e 100644
--- a/test/cctest/cctest.gyp
+++ b/test/cctest/cctest.gyp
@@ -82,6 +82,7 @@
         'compiler/test-run-stackcheck.cc',
         'compiler/test-run-stubs.cc',
         'compiler/test-run-variables.cc',
+        'compiler/test-run-wasm.cc',
         'compiler/test-simplified-lowering.cc',
         'cctest.cc',
         'gay-fixed.cc',
