diff --git a/test/cctest/cctest.gyp b/test/cctest/cctest.gyp
index b58aee9..fda8847 100644
--- a/test/cctest/cctest.gyp
+++ b/test/cctest/cctest.gyp
@@ -164,6 +164,8 @@
         'test-weakmaps.cc',
         'test-weaksets.cc',
         'trace-extension.cc',
+        'wasm/test-run-wasm.cc',
+        'wasm/test-run-wasm-module.cc',
       ],
       'conditions': [
         ['v8_target_arch=="ia32"', {
