diff --git a/src/d8.cc b/src/d8.cc
index a56d029..b6b7e71 100644
--- a/src/d8.cc
+++ b/src/d8.cc
@@ -51,6 +51,8 @@
 #include "src/v8.h"
 #endif  // !V8_SHARED
 
+#include "src/wasm/wasm-js.h"
+
 #ifdef V8_USE_EXTERNAL_STARTUP_DATA
 #include "src/startup-data-util.h"
 #endif  // V8_USE_EXTERNAL_STARTUP_DATA
@@ -1285,6 +1287,9 @@ Local<ObjectTemplate> Shell::CreateGlobalTemplate(Isolate* isolate) {
           .ToLocalChecked(),
       os_templ);
 
+  // Install WASM API.
+  WasmJs::Install(isolate, global_template);
+
   return global_template;
 }
 
