diff --git a/src/flag-definitions.h b/src/flag-definitions.h
index aebf72c..98184cd 100644
--- a/src/flag-definitions.h
+++ b/src/flag-definitions.h
@@ -446,6 +446,12 @@ DEFINE_BOOL(turbo_preserve_shared_code, false, "keep context-independent code")
 DEFINE_INT(typed_array_max_size_in_heap, 64,
            "threshold for in-heap typed array")
 
+// Flags for native WebAssembly.
+DEFINE_BOOL(trace_wasm_decoder, false, "trace decoding of wasm code")
+DEFINE_BOOL(trace_wasm_compiler, false, "trace compiling of wasm code")
+DEFINE_BOOL(wasm_break_on_decoder_error, false,
+            "debug break when wasm decoder encounters an error")
+
 // Profiler flags.
 DEFINE_INT(frame_count, 1, "number of stack frames inspected by the profiler")
 // 0x1800 fits in the immediate field of an ARM instruction.
