diff --git a/test/unittests/unittests.gyp b/test/unittests/unittests.gyp
index 64c4b10..78e8aee 100644
--- a/test/unittests/unittests.gyp
+++ b/test/unittests/unittests.gyp
@@ -94,6 +94,8 @@
         'run-all-unittests.cc',
         'test-utils.h',
         'test-utils.cc',
+        'wasm/decoder-unittest.cc',
+        'wasm/wasm-macro-gen-unittest.cc',
         '../../src/startup-data-util.h',
         '../../src/startup-data-util.cc'
       ],
