diff --git a/tools/gyp/v8.gyp b/tools/gyp/v8.gyp
index 0ac43de..cbb5481 100644
--- a/tools/gyp/v8.gyp
+++ b/tools/gyp/v8.gyp
@@ -988,6 +988,20 @@
         '../../src/version.h',
         '../../src/vm-state-inl.h',
         '../../src/vm-state.h',
+        '../../src/wasm/decoder.cc',
+        '../../src/wasm/decoder.h',
+        '../../src/wasm/tf-builder.h',
+        '../../src/wasm/tf-builder.cc',
+        '../../src/wasm/wasm-js.cc',
+        '../../src/wasm/wasm-js.h',
+        '../../src/wasm/wasm-linkage.cc',
+        '../../src/wasm/wasm-macro-gen.h',
+        '../../src/wasm/wasm-module.cc',
+        '../../src/wasm/wasm-module.h',
+        '../../src/wasm/wasm-opcodes.cc',
+        '../../src/wasm/wasm-opcodes.h',
+        '../../src/wasm/wasm-result.cc',
+        '../../src/wasm/wasm-result.h',
         '../../src/zone.cc',
         '../../src/zone.h',
         '../../src/zone-allocator.h',
